R/importWaterML1.r

```diff
@@ -97,356 +95,194 @@
 #' tzURL <- constructNWISURL("04027000", c("00300","63680"), "2011-11-05", "2011-11-07","uv")
 #' tzIssue <- importWaterML1(tzURL, TRUE, "America/Chicago")
 #'
-#' 
+#' #raw XML
+#' url <- constructNWISURL(service = 'dv', siteNumber = '02319300', parameterCd = "00060", 
+#'                          startDate = "2014-01-01", endDate = "2014-01-01")
+#' raw <- content(GET(url), as = 'raw')
+#' rawParsed <- importWaterML1(raw)
 #' }
 #' filePath <- system.file("extdata", package="dataRetrieval")
 #' fileName <- "WaterML1Example.xml"
 #' fullPath <- file.path(filePath, fileName)
-#' imporFile <- importWaterML1(fullPath,TRUE)
+#' importFile <- importWaterML1(fullPath,TRUE)
 #'
-importWaterML1 <- function(obs_url,asDateTime=FALSE, tz=""){
-  
-  if(file.exists(obs_url)){
-    rawData <- obs_url
-    returnedDoc <- xmlTreeParse(rawData, getDTD = FALSE, useInternalNodes = TRUE)
+
+importWaterML1 <- function(input,asDateTime=FALSE, tz=""){
```

Comment:
Even though I think it's good to let the `obs_url` take in a raw XML....I think it might be a bad idea to change the name of the argument. If people have code specified as:
```
importWaterML1(obs_url=fullPath,TRUE)
```
it's going to break scripts. So, can we keep the new functionality but change the name back?